library/pcs_property.py

```diff
@@ -130,13 +144,18 @@ def run_module():
         # we are stripping last line as they doesn't contain properties
         for row in out.split('\n')[0:-1]:
             # based on row we see the section to either cluster or node properties
-            if row == 'Cluster Properties:':
+            if row.startswith('Cluster Properties:'):
```

Comment:
I found the following differences that I had not noticed when I created the Issue.

pcs 0.10
```shell
$ sudo pcs property config
Cluster Properties:
 cluster-infrastructure: corosync
,,,
```
pcs 0.11
```
$ sudo pcs property config
Cluster Properties: cib-bootstrap-options
  cluster-infrastructure=corosync
...